BLE_GattServer_CharacteristicUpdates/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2020 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+
+set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/mbed-os CACHE INTERNAL "")
+set(MBED_CONFIG_PATH ${CMAKE_CURRENT_SOURCE_DIR}/.mbedbuild CACHE INTERNAL "")
+set(APP_TARGET BLE_GattServer_CharacteristicUpdates)
+
+include(${MBED_PATH}/tools/cmake/app.cmake)
+
+add_subdirectory(${MBED_PATH})
+
+add_subdirectory(mbed-os-ble-utils)
+
+add_executable(${APP_TARGET})
+
+mbed_configure_app_target(${APP_TARGET})
+
+mbed_set_mbed_target_linker_script(${APP_TARGET})
+
+project(${APP_TARGET})
+
+target_include_directories(${APP_TARGET}
+    PRIVATE
+        ./source
+)
+
+target_sources(${APP_TARGET}
+    PRIVATE
+        source/main.cpp
+)
+
+target_link_libraries(${APP_TARGET}
+    PRIVATE
+        mbed-os
+        mbed-events
+        mbed-ble
+        mbed-ble-cordio
+        mbed-ble-blue_nrg
+        mbed-os-ble-utils
```

Comment:
While we have this in here, I assue this example should have only `mbed-ble` + `mbed-ble-utils`. the rest should be added via dependencies, correct?

`mbed-ble-utils` rename needed, we do not need `mbed-os` prefix?